doc/ug/conf.py

```diff
@@ -12,6 +12,16 @@
 # serve to show the default.
 
 import sys, os
+import datetime
+
+# Gets version directly from the code
+try:
+    import espressopp
```

Comment:
I see, in #43 we could make the `ug` target depend on the espresso++ model and set the environment correctly like you suggested.